tools/benchmark.py

```diff
@@ -78,6 +82,22 @@ def main(argv):
             "min": data["min"],
             "max": data["max"]
         }
+
+    # Test only on linux
+    if "linux" in platform:
+        for [name, test_args] in thread_count_tests:
```

Comment:
Can you make the this whole block into its own function ? (So it can be tested more easily )

Reply:
When you abstract this into its own function, it would be extremely helpful if you added `tools/benchmark_test.py` and called your new function in it.

Follow the example here: https://github.com/denoland/deno/blob/17a7b03d1b86b3519e611aa855f561352075a954/tools/test.py#L33-L35